Misc/NEWS.d/next/Tests/2018-08-25-13-28-18.bpo-34347.IsRDPB.rst

```diff
@@ -0,0 +1,2 @@
+Fix `test_utf8_mode.test_cmd_line` for AIX
```

Comment:
I suggest:

```
Fix ``test_utf8_mode.test_cmd_line`` for AIX. Patch by M. Felt 
```

I would write "on AIX", but English is not my first language, so I kept your phrasing :-)

Reply:
on is perhaps the USA way, for is likely the impact of my second language.

I'll add the back-quotes.